bricksrc/equipment.py

```diff
@@ -545,56 +548,19 @@
         # - is any specific kind of switch e.g. PoESwitch implied here?
         # Video_Wall (or should this be in a separate classification with displays and monitors?)
     },
-    "Safety": {
```

Comment:
@epaulson I didn't mean to "remove" `Safety` here. I want this subtree to be independent to `Security_Equipment` as we discussed last time. I believe this subtree is still useful.

Reply:
Yes, I'll add it in a new PR once this one is finished.

Reply:
@epaulson I don't see Safety_Equipment in the recent commit yet. Do you want to merge this without it now?